swupd/rename.go

```diff
@@ -188,7 +188,7 @@ func fixupStatFields(needed []*File, m *Manifest, c *config) error {
 			bundleChroot = filepath.Join(c.imageBase, fmt.Sprint(m.Header.Previous), "full")
 		}
 		path := filepath.Join(bundleChroot, needed[i].Name)
-		fi, err := os.Stat(path)
+		fi, err := os.Lstat(path)
```

Comment:
I suspect that we need more logic around this than just swapping Stat to Lstat. My thought is that it doesn't make sense to rename symbolic links, the chances are that they are too small to be worth renaming.

Reply:
I agree, but to do that we have to persist the type flag for deleted files. The reason this link is sticking around and not cleaned up by trimRenamed is because it is deleted at that version and the file type is lost.

This is a quick fix for now to fix my testing.